### DISCUSSION TITLE
skybox change scale

### DISCUSSION THREAD

**Discussion by r0zbeh** [google/model-viewer#4905] [Q&A] [answered]
### Description

change model scale compare to Ground-projected Skybox/ i need to increase model scale but when i maximize model or use scale="2 2 2 " ground twists and turns 

how i can increase the model scale in skybox without deforme the skybox

![scale](https://github.com/user-attachments/assets/25e2d3f7-9ff8-4059-b876-d96c40be11bd)

### Version

- model-viewer: v3.5

### Browser Affected

ALL

### OS

All

### AR

- [ x] WebXR
- [x ] SceneViewer
- [x ] QuickLook

### Code 
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <script
      type="module"
      src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.5.0/model-viewer.min.js"
    ></script>
    <title>AKSTUDIO</title>
    <link rel="shortcut icon" type="image/jpg" href="./logo.jpg" />
  </head>
  <body>
    <model-viewer 
    id="viewer"
    camera-controls 
    style="width: 800px; height: 600px;"
     touch-action="pan-y"
      disable-pan 
      skybox-image="https://cdn.polyhaven.com/asset_img/primary/autumn_hockey.png"
      skybox-height="17.5m" 
      shadow-intensity="2" 
      scale="2 2 2"
      camera-orbit="auto auto 80m"
      min-camera-orbit="auto 0deg 40m"
      max-camera-orbit="auto 90deg 80m" 
      alt="A 3D astronaut model depicted within a forest" 
      src="red-div-2--.glb"
      field-of-view="90deg"
      >
    </model-viewer>

  </body>
  <script>

  </script>
</html>
```

**Comment by danilokrulj** [comment]
Did you try increasing the model scale in a 3D software (like Blender for example)?
If not, did you try increasing the model scale through code instead of the modelviewer scale parameter?

I changed model scale in some old code like this (probably not the best way but you can give it a try):

const modelViewer = document.querySelector("model-viewer#mw");
const sceneSym = Object.getOwnPropertySymbols(mw).find(x => x.description === "scene");
modelViewer[sceneSym].children[0].scale.x = 2;
modelViewer[sceneSym].children[0].scale.y = 2;
modelViewer[sceneSym].children[0].scale.z = 2;


Also, try playing arround with skybox-height, i tested with that hdr you are using and height seems to have a big effect on how skybox is rendered.

**Comment by r0zbeh** [comment]
### Did you try increasing the model scale in a 3D software (like Blender for example)?
yes , test but same problem !!
###  skybox-height 
When I use it, the distortion problem is solved, but the scale effect is lost and the volume is seems reduced again

### Does anyone have a solution?

**Comment by elalish** [comment]
Hmm, that `scale` thing sure looks like a bug. But if you want your skybox to look decent for a large size model, you need a skybox that's taken from a greater height. I would recommend having your camera ball about as high up as this house is tall, then set `skybox-height` to match that.

**Comment by r0zbeh** [comment]
### skybox-height - as say it in previous comment !!!!

When I use it, the distortion problem is solved, but the scale effect is lost and the volume is seems reduced again

![skybox2](https://github.com/user-attachments/assets/08dcfa98-7059-4215-9a32-83bf81276354)

**Comment by r0zbeh** [comment]
i need increase scale of house compare to skybox(trees , ...) 

### problem is when i increase the object scale , skyroom is grow too!!!!

![scal2](https://github.com/user-attachments/assets/bfcac2d1-d4f4-4bed-a4b4-0cc780ea3e05)

**Comment by elalish** [comment, marked as answer]
>you need a skybox that's taken from a greater height.

I mean it - your skybox is itself small. You have to use a different image - taken for higher up for it to look believable; I can't fix that with geometry or rendering.
